data/remnant.txt

```diff
@@ -131,12 +192,18 @@ phrase "remnant"
 		"Lost"
 		"Changed"
 		"Alive"
-
+		"Restored"
+		"Preserved"
+		"Renewed"
+		"Reborn"
+		"Ascending"
+		"Soaring"
+		"Enduring"
 
 
 shipyard "Remnant"
 	"Starling"
-	"Albatross"
```

Comment:
Agreed. That would be consistent with how every other race does it. However MZ saw fit to only give the Remnant a single license. If someone happens to create the graphics for a new license, I'll be happy to change things around to reflect that.

Reply:
Although that being said, I'm inclined to split it up into regular ships and capital ships instead of civilian vs military. In a society as small and tight as the Remnant, the distinction between what size ship one is qualified to fly would be much more significant than whether or not the ship is primarily transportation or primarily combat.